frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -0,0 +1,332 @@
+      <a href="https://docs.webknossos.org" target="_blank" rel="noreferrer">
+        user documentation...
+      </a>
+    </Hint>
+  );
+
+  const wklibsInitCode = `import webknossos as wk
+
+with wk.webknossos_context(token="MY_TOKEN"):
+    dataset = wk.Dataset.download(
+        dataset_name_or_url="dataset_name",
+        organization_id="my_organization",
+    )
+`;
+
+  return (
+    <Modal
+      title="Download this Annotation"
+      visible={isVisible}
+      width={600}
+      okText="Ok"
+      onOk={handleOk}
+      onCancel={onOk}
+      style={{ overflow: "visible" }}
+    >
+      <Tabs onChange={handleTabChange} type="card">
+        <TabPane tab="Download" key="1">
+          <Row>
+            <Col span={24}>
+              <Hint
+                style={{
+                  margin: "6px 12px",
+                }}
+              >
+                {messages["annotation.download"]}
+              </Hint>
+            </Col>
+          </Row>
+          <Divider
+            style={{
+              margin: "18px 0",
+            }}
+          >
+            Options
+          </Divider>
+          {maybeShowWarning()}
+          <Row>
+            <Col
+              span={9}
+              style={{
+                lineHeight: "20px",
+                padding: "5px 12px",
+              }}
+            >
+              Select the data you would like to download.
+            </Col>
+            <Col span={15}>
+              <CheckboxGroup onChange={handleCheckboxChange} defaultValue={selection}>
+                <Checkbox style={checkboxStyle} value="Volume">
+                  Volume as WKW
+                </Checkbox>
+                <Hint
+                  style={{
+                    marginLeft: 24,
+                    marginBottom: 12,
+                  }}
+                >
+                  Download a zip folder containing WKW files.
+                </Hint>
+
+                <Checkbox style={checkboxStyle} value="Skeleton">
+                  Skeleton as NML
+                </Checkbox>
+                <Hint
+                  style={{
+                    marginLeft: 24,
+                    marginBottom: 12,
+                  }}
+                >
+                  Download a zip folder containing NML files.
+                </Hint>
+
+                <Checkbox style={checkboxStyle} value="Fallback">
+                  Fallback Layers
+                </Checkbox>
+                <Hint
+                  style={{
+                    marginLeft: 24,
+                    marginBottom: 12,
+                  }}
+                >
+                  Download a zip folder containing fallback layers as WKW files.
+                </Hint>
+              </CheckboxGroup>
+            </Col>
+          </Row>
+          <Divider
+            style={{
+              margin: "18px 0",
+            }}
+          />
+          {moreInfoHint}
+        </TabPane>
+
+        <TabPane tab="Export" key="2">
```

Comment:
Why not name it `TIFF Export` to make this clear without having to read the small print first.